types/crew.model.ts

```diff
@@ -0,0 +1,64 @@
+import { Club } from './club.model';
+import { DisplayStatus } from './display-status.model';
+import { Event } from './event.model';
+import { Result } from './result.model';
+import { Rower } from './rower.model';
+
+export interface Crew {
+	id: string;
+	club_id: string;
+	regatta_id: string;
+	event_id: string;
+	sub_event_id: null; // Wat is een sub_event?
+	subEventCode: string;
+	knrb_num: number;
+	knrb_sequence: number; // Wat is de sequence?
```

Comment:
Bij het inschrijven bij de wedstrijd krijg je bijvoorbeeld:

Proteus 1
Proteus 2
ect

De sequence is dit nummer.

Reply:
Maar er is ook een 'sequence' hier? Is dat dan hetzelfde?

Reply:
knrb_sequence komt uit de import. Deze veranderd niet maar de er kunnen ploegen bij komen of velden samenvoegen

Reply:
Kan dat dan niet worden samengevoegd of 'berekend' door cake?

Reply:
Samengevoegd kan niet en berekend kan ook niet omdat je de volgorde van de ploegen overhoopt gooit bij de loting

Reply:
Sequence mag dan ook niet worden aangepast door de wl en hoeft niet getoond te worden